code/api/queries.md

```diff
@@ -0,0 +1,51 @@
+Get all user objects:
+
+```
+query {
+  users {
+    id
+    name
+    email
+    password
+    role
+    styleSummary
+    createdAt
+    updatedAt
+  }
+}
+```
+
+Get a single user object by ID:
+```
+query {
+  user(id: <integer>) {
+    id
+    name
+    email
+    password
+    role
+    styleSummary
+    createdAt
+    updatedAt
+  }
+}
+```
+
+Update a users styleSummary with style survey form data:
+```
+mutation {
+  userStyleSummaryUpdate(id: <integer>, formData: <form data>)
+}
+```
+
+---
+Form data parameter assumed to be in following format when sent with styleSummary mutation:
+```
+{
+  "tops": "punk",
+  "pants": "comfy",
+  "dresses": "punk",
+  "shoes": "punk",
+  "accessories": "prep"
+}
+```
```

Comment:
Is this the file you built just to show structure of query? We may want to remove (or add to gitignore) this afternoon when we finalize project, as this is really more a dev tool for us.

Reply:
I agree, this notes file may not need to be in our final deployed project

Reply:
Update: this markdown file in final PR